File: C++/First Steps in Coding - Basics of C++/First steps/Exercises/Food Delivery.cpp
#include <iostream>
#include <cstdlib>
using namespace std;

int main() {
	double chickenMenuPrice = 10.35, fishMenuPrice=12.40, vegetarianMenuPrice = 8.15, dessertPrice, delivery = 2.50, cost;
	int amountChickenMenu, amountFishMenu, amountVegetarianMenu;
	cin >> amountChickenMenu; cin >> amountFishMenu; cin >> amountVegetarianMenu;
	cost = (amountChickenMenu * chickenMenuPrice) + (amountFishMenu * fishMenuPrice) + (amountVegetarianMenu * vegetarianMenuPrice);
	dessertPrice = cost * 0.2;
	cost += dessertPrice + delivery;
	cout << cost;
}

File: Java/First steps in coding/Exercises/07. Food Delivery/FoodDelivery.java
import java.util.Scanner;
public class FoodDelivery {
    public static void main(String[] args){
        //prices
        double chickenMenuPrice = 10.35;
        double fishMenuPrice = 12.40;
        double vegetarianMenuPrice = 8.15;
        double dessertPrice = 0; //20% of total sum, delivery excluded
        double deliveryPrice = 2.50;

        Scanner input = new Scanner(System.in);

        int chickenMenus = Integer.parseInt(input.nextLine());
        int fishMenus = Integer.parseInt(input.nextLine());
        int vegetarianMenus = Integer.parseInt(input.nextLine());

        double orderTotal = ((double)chickenMenus * chickenMenuPrice) + ((double)fishMenus * fishMenuPrice) + ((double)vegetarianMenus * vegetarianMenuPrice);
        dessertPrice = orderTotal * 0.2;
        orderTotal += dessertPrice + deliveryPrice;

        System.out.println(orderTotal);
    }
}
